utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/json/ToJsonEncoder.scala

```diff
@@ -23,73 +22,29 @@ case class ToJsonEncoder(
     highPriority: PartialFunction[Any, Json] = Map()
 ) {
 
-  private val safeString     = safeJson[String](fromString)
```

Comment:
It seems that it was already "safe" - null was the first variant checked in patter matching